plugin/init.lua

```diff
@@ -369,30 +391,102 @@ function pub.fuzzy_load(window, pane, callback, opts)
 		end
 	end
 
-	local function insert_choices(type, fmt)
-		for _, file in ipairs(wezterm.glob("*", pub.save_state_dir .. "/" .. type)) do
-			local label
+	local function file_datetime(folder, file)
+		-- Month names
+		local months = {
+			"Jan",
+			"Feb",
+			"Mar",
+			"Apr",
+			"May",
+			"Jun",
+			"Jul",
+			"Aug",
+			"Sep",
+			"Oct",
+			"Nov",
+			"Dec",
+		}
+		local result = ""
+		if is_windows then
+			local handle = io.popen("wmic datafile where name='" .. folder .. separator .. file("' get LastModified"))
+			if handle then
+				local stdout = handle:read("*a")
+				handle:close()
+
+				-- Convert the date format into "dd-mmm-yyyy HH:MM"
+
+				local year = stdout:sub(1, 4)
+				local month = stdout:sub(5, 6)
+				local day = stdout:sub(7, 8)
+				local hour = stdout:sub(9, 10)
+				local minute = stdout:sub(11, 12)
+
+				-- Format into "dd-mmm-yyyy hh:mm"
+				result = string.format(
+					"%02d-%s-%04d %02d:%02d",
+					tonumber(day),
+					months[tonumber(month)],
+					tonumber(year),
+					tonumber(hour),
+					tonumber(minute)
+				)
+			end
+		else
+			local handle = io.popen("date -r '" .. folder .. "' +'%d-%b-%Y %H:%M'")
```

Comment:
Should this be folder? When I run `date -r '<dir>' +'%d-%b-%Y %H:%M'` I get only the date of the directory, not the individual files.

Is this intended, I would think it should be each individual file that should be dated, but i might be wrong?

Reply:
I would think it should add the file, like so:
```suggestion
			local handle = io.popen("date -r '" .. folder .. seperator .. file .. "' +'%d-%b-%Y %H:%M'")
```